src/navigators/Navigator.tsx

```diff
@@ -132,11 +135,16 @@ class Navigator extends React.Component<Props, State> {
   };
 
   renderRoutes = () => {
-    const { isLoading, unlockKey } = this.props;
+    const { isLoading, unlockKey, isTcAccepted } = this.props;
     if (isLoading) {
       return null;
     }
 
+    if (!isTcAccepted) {
+      //@ts-ignore
```

Comment:
what is the problem here?

Reply:
https://pastebin.com/1jNZN8be this how I think TermsConditionsScreen should look like. So without navigation, as it doesn't have it. TermsConditionsScreen should be removed from RootNavigator

Reply:
in the next step, we need to implement some T&C in Settings, not reason remove I think

Reply:
probably we will do a different screen that would share common logic in the component that both screens will use

Reply:
that is the scenario till we don't know I will left and write todos in the next step

Reply:
still, it should be solved so that ts-ignore is not needed

Reply:
:) another scenario can solve this ts ignor

Reply:
so we should make a ts friendly solution for now and not increase technical debt.  If we will have another scenario in the future that I can make a review of it, right now I'm reviewing what I see and the typings is incorrect in this flow

Reply:
done